src/TextareaAutosize.js

```diff
@@ -3,9 +3,11 @@
  */
 
 import React from 'react';
-import emptyFunction from 'react/lib/emptyFunction';
+import ReactDOM from 'react-dom';
```

Comment:
`findDOMNode` depends on a bunch of stateful stuff so including it twice will blow up. This effectively makes it impossible for libraries to support both versions at once. That being said if someone using 0.13 was using webpack (or you provide a built bundle) then it could be made to work fine, with aliases